tools/openshift/backend.dc.yaml

```diff
@@ -78,12 +72,56 @@ objects:
                 limits:
                   cpu: '${MAX_CPU}'
                   memory: '${MAX_MEM}'
+              volumeMounts:
```

Comment:
This volumeMounts element isn't the same position as backend-tls.dc.yaml which could be confusing if we have to compare or make change to both files.

Reply:
Moved to the bottom of the image config in both files